checks/libs/wmi/types.py

```diff
@@ -0,0 +1,24 @@
+# stdlib
```

Comment:
Why `types.py` ? Also I am not sure it makes sense to have it in `checks/libs/wmi` as it's related to `checks/wmi.py` not the sampler (which is quite stand alone).